app/components/file_attachment_list/file_attachment_icon.js

```diff
@@ -86,13 +88,13 @@ export default class FileAttachmentIcon extends PureComponent {
 const styles = StyleSheet.create({
     fileIconWrapper: {
         alignItems: 'center',
-        backgroundColor: '#fff',
         justifyContent: 'center',
         borderTopLeftRadius: 2,
         borderBottomLeftRadius: 2,
     },
     icon: {
         borderTopLeftRadius: 2,
         borderBottomLeftRadius: 2,
+        backgroundColor: '#fff',
```

Comment:
Did you check with a theme enabled? This might add white background to the icon alone and not to the wrapper

Reply:
@sudheerDev you can see it in a dark theme here
![image](https://user-images.githubusercontent.com/6757047/48636781-99de6080-e9aa-11e8-8d00-e5802f2cfd48.png)